docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -225,38 +215,47 @@ CEPH TEST SUMMARY: Suite contains 812, ran 387 tests, Passed: 387, Skipped: 0, F
 Following the 'Run a Single Ceph S3 Test' steps.
 
 ### 1) Test Pass
-For example: `s3tests_boto3.functional.test_s3:test_basic_key_count`
+For example: `s3tests_boto3/functional/test_s3.py::test_basic_key_count`
 ```
-bash-4.4$ S3TEST_CONF=src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf ./src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/bin/nosetests s3tests_boto3.functional.test_s3:test_basic_key_count
-.
-----------------------------------------------------------------------
-Ran 1 test in 4.914s
+bash-4.4$ S3TEST_CONF=$PWD/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- --disable-pytest-warnings $PWD/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py::test_account_usage
+
+...
+
+====================================================================================== 1 passed, 1 warning in 15.71s =======================================================================================
+_________________________________________________________________________________________________ summary __________________________________________________________________________________________________
+  py: commands succeeded
+  congratulations :)
+
```

Comment:
I think we should add a comment about the warnings that we have, for example:
`WARNING: could not copy distfile to //.tox/distshare`

Since the output has been changed dramatically I would attach also an image and add those explanations (and not just a copy of the lines as we have today).